### PR TITLE
included the info to get the device ID and register that in TP portal…

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**
included the info to get the device ID and register that in TP portal, rearranged content for macOS and Windows using tabs.
 @retaric  I have some redundant links in the side bar, will remove them once you are ok with the new restructured content as tabs.

### CODE REVIEW

File: prerequisites-for-onboarding.md [hunk 1]
```diff
@@ -1,53 +1,341 @@
-# Prerequisites
+# Prerequisites for SEED
 <div class="warn">
 
-Things to take note before you proceed with the onboarding:
+Take note of the following before you proceed with the SEED onboarding:
+
+  - You can onboard only a non-GSIB or a non-DWP device to SEED.
+
+  - If you are a public officer, after you enrol in Intune, you need to register your Intune device ID to complete the SEED onboarding for your device.
```
Comment (retaric) [inline prerequisites-for-onboarding.md:8]: Let's change "register" to "submit" for consistency.

File: _sidebar.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+* [Post onboarding verification for GCC 1.0 users](seed-post-onboarding-verification-for-gcc-1.0)
+* [Offboard device from SEED](offboard-device-from-seed)
 * macOS users
   * [Onboard](seed-onboarding-instructions-for-macos)
+  * [Post onboarding instructions](post-onboarding-instructions-for-macos)
   * [Offboard](seed-offboarding-instructions-for-macos)
 * Windows users  
   * [Onboard](seed-onboarding-instructions-windows)
+  * [Post onboarding instructions](post-onboarding-instructions-for-windows)
   * [Offboard](seed-offboarding-instructions-for-windows)
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply _sidebar.md]: i have moved them now.

File: onboard-device-to-seed.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+  10. You will see the success message. Click **Done**.
+
+  <kbd>![all-set](images/onboarding-for-macos/all-set-2.png)</kbd>
+
+
+</details>
+
+<details>
+<summary>b. If you are a public officer, register the Microsoft Intune device ID for your macOS device</summary>
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply onboard-device-to-seed.md:73]: changed

File: onboard-device-to-seed.md [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+
+This step is applicable only for public officers to get the required applications and device configurations on their device.
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply onboard-device-to-seed.md:75]: I have added a note after line 75:
?> If you are a public officer; your TechPass ID will be your official email address such as *your_name<span>@</span>agency.gov.sg* or *your_name<span>@</span>tech.gov.sg*. Ignore this step if your TechPass ID format is *your_name<span>@</span>techpass.gov.sg*.

File: onboard-device-to-seed.md [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+?> Vendors will see **TechPass** instead of **SG Govt M365**.
+
+<kbd>![managed-by-sg-govt-m365](images/onboarding-instructions-for-windows/managed-by-sg-govt-m365.png ':size=600')</kbd>
+
+Now your device is enrolled in Microsoft Intune. If you are a vendor or contractor, all the required applications and device configurations are available on your device.
+</details>
+
+<details><summary>b. If you are a public officer, register the Microsoft Intune device ID for your Windows device</summary>
+
+This step is applicable only for public officers to get the required applications and device configurations on their device.
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply onboard-device-to-seed.md:142]: added the same note: 

?> If you are a public officer; your TechPass ID will be your official email address such as *your_name<span>@</span>agency.gov.sg* or *your_name<span>@</span>tech.gov.sg*. Ignore this step if your TechPass ID format is *your_name<span>@</span>techpass.gov.sg*.

File: onboard-device-to-seed.md [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+$certs = (Get-ChildItem -path Cert:\CurrentUser\My);
+$output = "null";
+foreach ($cert in $certs) {
+    if ($cert.Issuer -eq "CN=Microsoft Intune MDM Device CA") {
+        $output = $cert.Subject.Split("=")[1];
+        break;
+    }
+}
+Write-Output $output;
+```
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply onboard-device-to-seed.md:157]: done

File: prerequisites-for-onboarding.md [hunk 7]
```diff
@@ -1,53 +1,341 @@
...
+1. [Request for TechPass account and SEED provisioning from your project manager or reporting officer](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-vendors-to-techpass?id=step-1-get-a-techpass-account-and-seed-licence-for-vendors-or-contractors).
+2. [Sign in to TechPass](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-vendors-to-techpass?id=step-2-first-time-sign-in-using-initial-password).
+3. [Set up MFA for your TechPass account](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-vendors-to-techpass?id=step-3-configure-and-verify-mfa-for-techpass-account).
+4. [Reset initial password](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-vendors-to-techpass?id=step-4-reset-your-initial-password).
+5. [Accept TechPass and MDM policies](https://docs.developer.tech.gov.sg/docs/techpass-user-guide/#/onboard-vendors-to-techpass?id=step-5-accept-terms-of-use-privacy-policy-and-mobile-device-management-acceptable-use-policy).
+
+<!-- tabs:end -->
+
+### SEED provisioning
+Once SEED is provisioned to you, you will receive an email with instructions on how to onboard your device to SEED.
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply prerequisites-for-onboarding.md:54]: I believe this is applicable to users who are requesting for SEED when they sign up for a TP account.

If you had requested SEED provisioning while signing up for your TechPass account, SEED will be provisioned to you and you will receive an email with instructions on how to onboard your device to SEED. 

?> If you are a public officer, who already has a TechPass account and needs SEED provisioning, go to your profile page on the [TechPass portal](http://portal.techpass.gov.sg/) and request for SEED provisioning.

File: prerequisites-for-onboarding.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply prerequisites-for-onboarding.md:8]: done

File: offboard-device-from-seed.md [hunk 8]
```diff
@@ -0,0 +1,177 @@
+# Offboard your device from SEED
+
+This page tells you how to onboard your device to SEED.
```
[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply offboard-device-from-seed.md:3]: thank you for spotting this.

[the remark this answers is not in the document]
Comment (SenthilKumarSivakami) [reply offboard-device-from-seed.md]: @retaric , Pls take a look at the revised sections

**Review by retaric — APPROVED** [review]

**Pull request merged**